Cython/Compiler/Pipeline.py

```diff
@@ -253,9 +253,11 @@ def create_pyx_pipeline(context, options, result, py=False, exclude_classes=()):
         [parse_stage_factory(context)],
         create_pipeline(context, mode, exclude_classes=exclude_classes),
         test_support,
-        [inject_pxd_code_stage_factory(context),
-         inject_utility_code_stage_factory(context),
-         abort_on_errors],
+        [
+            inject_pxd_code_stage_factory(context),
+            inject_utility_code_stage_factory(context),
+            abort_on_errors
+        ],
```

Comment:
This change, although reasonable, is unrelated to this PR.

Reply:
Is it bad that the codebase is getting better? Even if it is not related .. after all

..

Well, should I remove these changes too?

Reply:
I don't mind the code base getting better. I do mind making `git blame` refer to unrelated changes and PRs.

Reply:
[resolved](https://github.com/cython/cython/pull/4241/commits/615dad02b139c1b9f0ce0288c9d570ab1d51f90a)